netg@DESKTOP-EBH0I8A:~/devnet$ cd autobundle-nov24/
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24$ cd dict/
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
<class 'dict'>
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, 100, 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': 'Hello world 2'}, 'data4': [{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, 100, 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': 'Hello world 2'}, 'data4': [{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, 100, 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': 'Hello world 2'}, 'data4': [{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 5
    "data3": {"key1": "Hello world 1",
             ^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: invalid syntax. Perhaps you forgot a comma?
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, 100, 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': 'Hello world 2'}, 'data4': [{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, 100, 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': 'Hello world 2'}, 'data4': [{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 7
    "key3": "Hello world 3",
          ^
SyntaxError: invalid syntax
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_keys(['data1', 'data2', 'data3', 'data4'])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
This is the second class for dictionary
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Traceback (most recent call last):
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 12, in <module>
    print(dict_data["data11"])
KeyError: 'data11'
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
This is the second class for dictionary
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
None
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Key not found
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
['Ravi', 'Prince', 10, 100, 200, 20.3]
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Prince
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Traceback (most recent call last):
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 12, in <module>
    print(dict_data("data2")[1])
TypeError: 'dict' object is not callable
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Prince
Prince
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Prince
Traceback (most recent call last):
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 13, in <module>
    print(dict_data.get("data22")[1])
TypeError: 'NoneType' object is not subscriptable
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'key1': 'Hello world 1', 'key2': 'Hello world 2', 'key3': 'Hello world 3'}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Hello world 2
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
[{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
b2
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Traceback (most recent call last):
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 18, in <module>
    print(dict_data["data2"][3]["b"])
TypeError: list indices must be integers or slices, not str
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
b22
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}], 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': 'b22'}, {'c': 'c1'}]], 200, 20.3], 'key3': 'Hello world 3'}, 'data4': [{'a': 'a1'}, {'b': 'b2'}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Traceback (most recent call last):
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 16, in <module>
    print(dict_data["data3"]["key2"][3][2]["c"][4])
IndexError: list index out of range
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Traceback (most recent call last):
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 16, in <module>
    print(dict_data["data3"]["key2"][3][2]["c"][2])
IndexError: list index out of range
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
3
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'data1': 'This is the second class for dictionary', 'data2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}], 200, 20.3], 'data3': {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': 'b22'}, {'c': [1, 2, 3, 4]}]], 200, 20.3], 'key3': 'Hello world 3'}, 'data4': [{'a': 'a1'}, {'b': {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}]}, {'c': [1, 2, 3, 4]}]], 200, 20.3], 'key3': 'Hello world 3'}}, {'c': 'c1'}]}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ cd ..
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24$ git status
On branch main
Your branch is up to date with 'origin/main'.

Untracked files:
  (use "git add <file>..." to include in what will be committed)
        dict/dict_funda_2.py

nothing added to commit but untracked files present (use "git add" to track)
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24$ git add .
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24$ git commit -m "added dictionary data"
[main ffd3e49] added dictionary data
 1 file changed, 22 insertions(+)
 create mode 100644 dict/dict_funda_2.py
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24$ git push
Enumerating objects: 6, done.
Counting objects: 100% (6/6), done.
Delta compression using up to 8 threads
Compressing objects: 100% (4/4), done.
Writing objects: 100% (4/4), 641 bytes | 641.00 KiB/s, done.
Total 4 (delta 1), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (1/1), completed with 1 local object.
To https://github.com/siddiquee4you/autobundle-nov24.git
   705601a..ffd3e49  main -> main
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24$ cd dict/
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ cd dict/
bash: cd: dict/: No such file or directory
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_keys(['data1', 'data2', 'data3', 'data4'])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_values(['This is the second class for dictionary', ['Ravi', 'Prince', 10, [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}], 200, 20.3], {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': 'b22'}, {'c': [1, 2, 3, 4]}]], 200, 20.3], 'key3': 'Hello world 3'}, [{'a': 'a1'}, {'b': {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}]}, {'c': [1, 2, 3, 4]}]], 2000, 20.3], 'key3': 'Hello world 3'}}, {'c': 'c1'}]])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_keys(['data1', 'data2', 'data3', 'data4'])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
  File "/home/netg/devnet/autobundle-nov24/dict/dict_funda_2.py", line 15
    print(dict_data.["data1"])
                    ^
SyntaxError: invalid syntax
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
This is the second class for dictionary
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_keys(['data1', 'data2', 'data3', 'data4'])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
['Ravi', 'Prince', 10, [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}], 200, 20.3]
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': 'b22'}, {'c': [1, 2, 3, 4]}]], 200, 20.3], 'key3': 'Hello world 3'}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
[{'a': 'a1'}, {'b': {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}]}, {'c': [1, 2, 3, 4]}]], 2000, 20.3], 'key3': 'Hello world 3'}}, {'c': 'c1'}]
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'a': 'a1'}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'b': {'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}]}, {'c': [1, 2, 3, 4]}]], 2000, 20.3], 'key3': 'Hello world 3'}}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_keys(['b'])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
{'key1': 'Hello world 1', 'key2': ['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}]}, {'c': [1, 2, 3, 4]}]], 2000, 20.3], 'key3': 'Hello world 3'}
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
dict_keys(['key1', 'key2', 'key3'])
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
Hello world 1
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
['Ravi', 'Prince', 10, [{'a': 'a1'}, [{'b': [{'a': 'a1'}, {'b': 'b22'}, {'c': 'c1'}]}, {'c': [1, 2, 3, 4]}]], 2000, 20.3]
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ 
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
2000
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
20.3
netg@DESKTOP-EBH0I8A:~/devnet/autobundle-nov24/dict$ python3 dict_funda_2.py 
20
